cola/widgets/remote.py

```diff
@@ -3,10 +3,19 @@
 import time
 import os
 
+from cola.models import prefs
+
+from cola import resources
+
 from qtpy import QtGui
 from qtpy import QtWidgets
 from qtpy.QtCore import Qt
 
+try:
+    from notifypy import Notify
+except (ImportError, ModuleNotFoundError):
+    pass
```

Comment:
Instead of `pass` we should set `Notify = None`.

Better yet, let's not use `from ... import` but instead just use `import notifypy` and in the `except` here we would set `notifypy = None` instead. That's a little more natural.

Then, the `_emit_push_notification(...)` function can check `if notifypy is None:` rather than having to test against a Capitalized name.

Reply:
Also, sort the imports. `notifypy` should go before `qtpy` in that block of imports.